components/process.rst

```diff
@@ -128,6 +128,28 @@ environment variables using the second argument of the ``run()``,
     // On both Unix-like and Windows
     $process->run(null, ['MESSAGE' => 'Something to output']);
 
+Setting Environment Variables for Processes
+-------------------------------------------
+
+The constructor of the :class:`Symfony\\Component\\Process\\Process` class and
+all its methods related to executing processes (``run()``, ``mustRun()``,
```

Comment:
```suggestion
all of its methods, related to executing processes (``run()``, ``mustRun()``,
```

Reply:
I don't think this suggestion is correct. This doesn't apply to "all methods" ... but to "all methods which are related to executing processes". Do you still think we should reword the original to make it more clear?